lms/routes.py

```diff
@@ -31,3 +31,5 @@ def includeme(config):
     )
     config.add_route("canvas_api.files.via_url", "/api/canvas/files/{file_id}/via_url")
     config.add_route("lti_api.submissions.record", "/api/lti/submissions")
+
+    config.add_route("lti_api.result.record", "/api/lti/result", request_method="POST")
```

Comment:
I anticipate adding a second route handler for `request_method="GET"` later, to fetch existing grade.